### ISSUE TITLE
saved_change_to_tag_names? not working… 

### ISSUE THREAD

**Issue by downtroden** [pat/gutentag#49]
isn't this creating a tag_names attribute?

**Comment by pat** [comment]
There's a bit of explanation around this…

Early versions of Gutentag supported the changed state methods for `tag_names`, but Rails 4.2 changed the underlying logic behind this - and at the time, it seemed it was only useful for database columns. `tag_names` is obviously not a database column, so I had to drop support for Rails 4.2 onwards.

Your question prompted me to revisit and see if it was possible again in more recent versions of Rails. As it turns out, it _is_, and I've just spent a few hours getting it working - including Rails 4.2, and so these features are available in Gutentag for _every_ version of Rails it supports (3.2-5.2).

The commits are in the `master` branch - if you'd like to give it a shot, that'd be great:

```ruby
gem "gutentag", "~> 2.1",
  :git    => "https://github.com/pat/gutentag.git",
  :branch => "master"
```

I'll aim to get a gem release with these changes (likely v2.2.0) out in the next week or so.
